pubsub.go

```diff
@@ -500,22 +547,34 @@ loop:
 	}
 
 	if throttle {
-		log.Warningf("message validation throttled; dropping message from %s", src)
```

Comment:
why drop the warning?

Reply:
it gets logged on the return path, so this would be logging it twice.

Reply:
Basically the function doesn't do the dropping any more, just validation; so the logging has been moved to the caller.